.github/workflows/create-release-from-tag.yml

```diff
@@ -20,10 +20,12 @@ jobs:
         node-version: 18
         registry-url: 'https://registry.npmjs.org'
         cache: npm
+    - name: Install latest npm version
+      run: npm install -g npm@latest
```

Comment:
we don't need to be explicit about `npm@latest`, just installing `npm` will use the `latest`